server/app/views/questiontypes/PhoneQuestionFragment.html

```diff
@@ -2,8 +2,13 @@
   th:fragment="phone(question, questionRendererParams)"
   th:with="phoneQuestion=${question.createPhoneQuestion()},
              questionId=${'id-' + #strings.randomAlphanumeric(8)},
-             inputId=${'cf-phone-number-' +'id-' + #strings.randomAlphanumeric(8)}"
+             inputId=${'cf-phone-number-' + #strings.randomAlphanumeric(8)},
+             fieldErrors=${phoneQuestion.getValidationErrors().get(phoneQuestion.getPhoneNumberPath())},
+             questionErrors=${phoneQuestion.getValidationErrors().get(question.getContextualizedPath())},
+             hasErrors=${!phoneQuestion.getValidationErrors().isEmpty()}"
```

Comment:
question (blocking):  I notice that the errors show immediately on the phone number question when the user first starts the block.  This is different behavior than with the other types of single-input questions.  Are phone number questions supposed to be different that way?

Reply:
that's a bug. I've got an open issue for it here https://github.com/civiform/civiform/issues/7007 to make sure I'm only rendering the errors in the right instances